ui/basicTab/basicTab.pug

```diff
@@ -0,0 +1,58 @@
+include ../global/advantages.pug
+
+// Basic section 
+.basic.section
+  = '\n'
+  | Race 
```

Comment:
These probably should be labels or `p` elements. I'll rework basic tab to the new styling in follow ups